actions/install/src/install.ts

```diff
@@ -1,6 +1,7 @@
 import * as core from '@actions/core'
 import * as tc from '@actions/tool-cache'
 import * as github from '@actions/github'
+import { exec } from 'child_process'
```

Comment:
Let's change this out for GitHub's toolkit `exec` package: https://github.com/actions/toolkit/tree/main/packages/exec. It has better cross-platform support and provides async support. It's also easier to mock.